CHANGES.md

```diff
@@ -17,6 +17,7 @@ Release Notes.
 * Enhance Apache ShenYu (incubating) plugin: support trace `grpc`,`sofarpc`,`motan`,`tars` rpc proxy.
 * Add primary endpoint name to log events.
 * Fix Span not finished in gateway plugin when the gateway request timeout.
+* Fix discussions #9113, log4j2 grpc report missing traceId
```

Comment:
I think we need some update on the docs https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/application-toolkit-log4j-2.x/

Please update `application-toolkit-log4j-2.x.md` accordingly.

Reply:
```suggestion
* Support `-Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector` in gRPC log report.
```

Reply:
Notice #9113 is actually not working, because this is not in the same repository.

Reply:
Do I need to recreate an issues

Reply:
No, I mean the link.`#issue/discussion number` is only working as a link when PR and issue are in the same repo. And we don't put this in the change log. The changelog is related to a commit/PR already.

Reply:
ok